Makefile

```diff
@@ -140,6 +140,7 @@ user_run:
 		--hostname in-container \
 		--rm \
 		$(EXTRA_DOCKER_RUN_ARGS) \
+		--group-add stack \
```

Comment:
This won't work if the group doesn't exist on the host PC. I would replace it with the numeric value of the stack group created in the container. If you call groupadd with -g you can specify the numeric value of the GID in the container.

Reply:
I did some tests around this, and I think (at least for me) it only fails if the container doesn't have the group.

For example, using a stock Debian image with a group I don't have on my host:
```bash
$ grep thisGroupDoesNotExistOnHost /etc/group; echo $?
1

$ docker run -it --group-add thisGroupDoesNotExistOnHost debian:buster bash
docker: Error response from daemon: Unable to find group thisGroupDoesNotExistOnHost.
ERRO[0000] error waiting for container: context canceled 
```
Now if I make a Dockerfile like this:
```Dockerfile
FROM debian:buster

RUN groupadd thisGroupDoesNotExistOnHost
```
and then:
```bash
$ docker build -t debian_group .
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM debian:buster
 ---> 1b686a95ddbf
Step 2/2 : RUN groupadd thisGroupDoesNotExistOnHost
 ---> Running in d86fbdd936ae
Removing intermediate container d86fbdd936ae
 ---> c3c549367874
Successfully built c3c549367874
Successfully tagged debian_group:latest

$ docker run -it --group-add thisGroupDoesNotExistOnHost debian_group:latest bash
root@61f08d20a871:/# groups
root thisGroupDoesNotExistOnHost
root@61f08d20a871:/#
```

Does it work the same for you?

Reply:
this works. It looks like make user_camkes didn't execute the scripts/camkes.sh script but took some preexisting image which didn't contain the group. So all good, it was my fault :)